app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
+
+  def current_user
+    user.first
+  end
```

Comment:
In order to make current user method data available to all controllers, Kindly move the current_user method to your Application Controller. ✊ 
`app/controllers/application_controller.rb`